packages/cli/src/index.js

```diff
@@ -8,12 +8,15 @@
  */
 
 // gracefulify() has to be called before anything else runs
-require('graceful-fs').gracefulify(require('fs'));
+import fs from 'fs';
+import gracefulFs from 'graceful-fs';
 
-const cli = require('./cliEntry');
+gracefulFs.gracefulify(fs);
+
+const cli = require('./cliEntry').default;
 
 if (require.main === module) {
   cli.run();
 }
 
-module.exports = cli;
+export default cli;
```

Comment:
please revert this export change as it changes our public API for those using `require` syntax.
Instead of doing:
```js
const rncli = require('@react-native-community/cli');
```
They would have to do:
```js
const rncli = require('@react-native-community/cli').default;
```
because of Babel interop. The API however won't change for folks using `import` syntax, because it has this check built-in (both in Babel and TS afaik)